notebooks/utils/utils.py

```diff
@@ -179,3 +179,30 @@ def timeseries_and_history_comparison(casename, output_roots):
         else:
             print(f"Could not find time series for all variables in year {year:04}")
         print("----")
+
+
+################################################################################
+
+
+def dict_copy_vals(src, dst, keys):
+    for key in keys if type(keys) == list else [keys]:
+        if key in src:
+            dst[key] = src[key]
```

Comment:
Per my comment on the call to `dict_copy_vals()`, I wonder if we want some error-checking here. Maybe something like

```py3
if key in src:
   if key in dst:
      if dst[key] != src[key]:
         # raise exception
   else:
      dst[key] = src[key]
```

Given what I know about how the time series files are generated, I wouldn't expect this Exception to be triggered but it might help catch issues if future time series generation tools accidentally muck with these keys (I wouldn't be surprised if we have an `xarray`-based time series generation tool at some point, and we already know about issues with `xarray` and unexpected encoding in output)

Reply:
Good point. I'll add something like this, controlling the exception with an optional argument to dict_copy_vals that I'll add.

While working on this, I have noticed that our current timeseries generation tool adds _FillValue to the variable time, and this attribute doesn't exist in the original tavg files. I don't think this is ideal, but also don't think it worth the effort to avoid that and/or modify timeseries that have been generated. My invocation of `dict_copy_vals` above propagates that into the output of `gen_dataset`, if there are timeseries files being used.